### DISCUSSION TITLE
Clicking on an alert on "Recent changes" should open the alert graph

### DISCUSSION THREAD

**Discussion by nikobearrr** [grafana/grafana#37024] [Alerting]
**What would you like to be added**:

I would like to be able to click on the alert name on the "Recent changes" and open the alert dashboard/graph.

**Why is this needed**:
We have a "Recent changes" for all our alerts accross the whole system. I would like to be able to open an alert and check what has happened faster. If I am able to click on the alert in the "Recent changes" component that will save some time.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
